activerecord/lib/active_record/connection_adapters/abstract/schema_creation.rb

```diff
@@ -59,6 +59,10 @@ def visit_TableDefinition(o)
             statements.concat(o.check_constraints.map { |chk| accept chk })
           end
 
+          if supports_exclusion_constraints?
+            statements.concat(o.exclusion_constraints.map { |chk| accept chk })
```

Comment:
`chk` looks comes from the check constraint above, here `chk` should be `exc`, `ec` or  something like that.